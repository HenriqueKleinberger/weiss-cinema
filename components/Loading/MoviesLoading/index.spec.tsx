import React from 'react';
import { shallow } from 'enzyme';
import MovieLoading from './index';

describe('MovieLoading', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<MovieLoading />);
    expect(wrapper).toMatchSnapshot();
  });
});
