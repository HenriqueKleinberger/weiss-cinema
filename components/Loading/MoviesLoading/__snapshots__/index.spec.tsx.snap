// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieLoading should match snapshot 1`] = `
<styled.div>
  <styled.div />
  <styled.div />
  <styled.div />
  <styled.div />
  <styled.div />
  <styled.div />
  <styled.div />
  <styled.div />
  <styled.div />
  <styled.div />
</styled.div>
`;
